.gitignore

```diff
@@ -1,5 +1,7 @@
+/.idea/
```

Comment:
This should be in your global gitignore file, but not here.

Reply:
thanks for the tip, done: https://github.com/arderyp/panther/commit/7124b9e85133b04cda0ead906b47024f56c0fea4